.github/workflows/fork-ci.yaml

```diff
@@ -0,0 +1,107 @@
+name: Fork Docker CI
+
+on:
+  push:
+    branches:
+    - network-operator-*
+    tags:
+    - network-operator-*
+
+jobs:
+  determine_docker_registry_and_tag:
+    runs-on: ubuntu-latest
```

Comment:
will it use only LTS version?

Reply:
[Documentation](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) states that `ubuntu-latest` currently uses 22.04.